.swiftformat

```diff
@@ -0,0 +1,17 @@
+# Formatting Options - Mimic Google style
+--indent 2
+--maxwidth 100
+--wrapparameters afterfirst
+
+# Disabled Rules
+
+# sortedImports is broken, sorting into the middle of the copyright notice.
+--disable sortedImports
```

Comment:
Note: I'm not sure that this is still an issue. I tried commenting it out locally, introduced an ordering issue, and it re-ordered correctly.

Reply:
I'd be fine to remove

Reply:
Done. Didn't see any issues in [`5190b43` (#13423)](https://github.com/firebase/firebase-ios-sdk/pull/13423/commits/5190b43d85f4150478952069f417c2dba0055a10) in the `check` [run](https://github.com/firebase/firebase-ios-sdk/actions/runs/10150161677/job/28066665430?pr=13423) on the entire repo.